tests/gtest/are_images_equal.cc

```diff
@@ -3,6 +3,7 @@
 // Compares two files and returns whether they are the same once decoded.
 
 #include <iostream>
+#include <string>
```

Comment:
This is actually a compilation error, not a warning:

libavif\tests\gtest\are_images_equal.cc(46): error C2039: 'stoi': is not a member of 'std'
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.31.31103\include\iostream(19): note: see declaration of 'std'
libavif\tests\gtest\are_images_equal.cc(46): error C3861: 'stoi': identifier not found